docs/assistant/ai-alert-triage.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[assistant-triage]]
+= Triage Alerts with Elastic AI Assistant
+Elastic AI Assistant can help you enhance and streamline your alert triage workflows by assessing multiple recent alerts in your environment, and helping you interpret an alert and its context. 
+
+When you view an alert in {elastic-sec}, details such as related documents, hosts, and users appear alongside a synopsis of the events that triggered the alert. This data provides a starting point for understanding a potential threat. AI Assistant can answer questions about this data and offer insights and actionable recommendations to remediate the issue.
+
+To enable AI Assistant to answer questions about alerts, you need to provide alert data as context for your prompts. You can either provide multiple alerts using the <<configure-ai-assistant, knowledge base>> feature, or provide individual alerts directly.
```

Comment:
nit: is it possible for the `knowledge base` links to jump to a KB-specific anchor? (I'm wondering if it's a quirk of the docs preview)

Reply:
Yes 100%. I am planning to do this but I have to merge the updates to the AI Assistant page before I can link to that section.